### PR TITLE
Changes for Finder classes

### LINKED ISSUE #64: filtering/ finding medicine, patient, doctor

### LINKED ISSUE #100: Linking the Finder classes to Manager class

### PR DESCRIPTION
**Pull request by wraineflores**
Create a new update branch as my previous PR keeps failing
Merge this PR first since this doesn't affect other classes
Closes #64 
Closes #100

### CODE REVIEW

**Comment by wraineflores** [comment]
Linked the 3 Finder classes with Manager class
Request for review

File: src/main/java/seedu/duke/helper/command/FindPatientCommand.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import seedu.duke.exception.HalpmiException;
+import seedu.duke.status.Status;
+
+public class FindPatientCommand extends Command {
+
+    public FindPatientCommand(String[] parameterArray) {
+        super(parameterArray);
+    }
+
+    public Status execute(List patientList) throws HalpmiException {
```
Comment (shxr3f) [inline src/main/java/seedu/duke/helper/command/FindPatientCommand.java:13]: So the find methods return an array, what do you do with this array?

File: src/main/java/seedu/duke/assets/MedicineList.java [hunk 2]
```diff
@@ -95,4 +98,59 @@ public void edit(String[] parameterArray) throws NotFoundException {
...
+        // Intentionally left blank
+    }
+
+    public void findBySpecialization(String[] parameters) {
+        // Intentionally left blank
+    }
+
+    public void findByDateAdmission(String[] parameters) {
+        // Intentionally left blank
+    }
```
Comment (shxr3f) [inline src/main/java/seedu/duke/assets/MedicineList.java:204]: All these are irrelevant to Medicine?

Comment (wraineflores) [reply src/main/java/seedu/duke/assets/MedicineList.java:204]: It's not but I need to put it there cos y'all implemented abstract in List so those that extended List.
Those "left blank" method essentially doesn't return anything. But can't remove it cos it's abstract subclass implementation

File: src/main/java/seedu/duke/helper/command/FindPatientCommand.java (hunk 1 above)
Comment (wraineflores) [reply src/main/java/seedu/duke/helper/command/FindPatientCommand.java:13]: In PatientList/DoctorList/MedicineList, it prints out the filtered Patient/Doctor/Medicine (zero to many) based on the attribute the user wants to filter it by. There are methods in the respective List classes

**Pull request merged**